deploy/template.yaml

```diff
@@ -182,6 +187,7 @@ Mappings:
       nodeOldSpaceLimit: "1792"
       nodeEnv: "production"
       languageToggle: true
+      outageBanner: true
```

Comment:
Can we have this off in production to start with, so it can be QA'd first?

Especially as we don't have the welsh content yet!

Reply:
yes I will update it thanks